tasks/16.html

```diff
@@ -12,7 +12,25 @@ <h1 class="global-heading">&nbsp;</h1>
 
         <main class="global-results">
             <style>
-                /* your styles here */
+                div:empty
+                {
+                    background: #ff6698;
+                }
+                
+                div.c1
```

Comment:
В селекторе класса можно не указывать тег
```suggestion
               .c1
```

Reply:
работает, только если добавить !important